### PR TITLE
change format labels in gui

### PR DESCRIPTION
**Pull request by darodi**
change format labels in gui

### CODE REVIEW

**Comment by darodi** [comment]
@axu2 

Tell me if this is OK for you, I'll merge it.

**Comment by axu2** [comment]
Screenshot would have been nice in future: 
image
Appreciate the formats dict.

Did you mean to use `KFX.png`?

File: kindlecomicconverter/KCC_gui.py [hunk 1]
```diff
@@ -918,6 +913,16 @@ def __init__(self, kccapp, kccwindow):
             if self.windowSize == '0x0':
                 MW.resize(500, 500)
 
+        self.formats = {  # text, icon, data/option_format
+            "MOBI/AZW3": {'icon': 'MOBI', 'format': 'MOBI'},
+            "EPUB": {'icon': 'EPUB', 'format': 'EPUB'},
+            "CBZ": {'icon': 'CBZ', 'format': 'CBZ'},
+            "EPUB (Calibre KFX)": {'icon': 'EPUB', 'format': 'KFX'},
```
Comment (axu2) [inline kindlecomicconverter/KCC_gui.py:920]: Should this be `KFX.png`?

Comment (darodi) [reply kindlecomicconverter/KCC_gui.py:920]: As you said, the result is not really a kfx file but an epub that needs to be converted to kfx in a second time. 
That's why I left an epub icon.

Comment (axu2) [reply kindlecomicconverter/KCC_gui.py:920]: Alright just confirming. LGTM

**Pull request merged**